content/docs/2.9/operate/prometheus.md

```diff
@@ -10,6 +10,7 @@ weight = 200
 
 The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080` at `/metrics`. The following metrics are being gathered:
 
+- `keda_operator_crd_totals` - Total number of KEDA custom resources per namespace for each resource.
```

Comment:
The current phrasing makes it feel like it's per resource instead of CRD
```suggestion
- `keda_operator_crd_totals` - Total number of KEDA custom resources per namespace for each custom resource type (CRD).
```

Reply:
Incorporated the suggestion. Also modified the metric name so that it's slightly more intuitive.